docs/utils/use-update-atom.mdx

```diff
@@ -4,6 +4,8 @@ title: useUpdateAtom
 
 Ref: https://github.com/pmndrs/jotai/issues/26
 
+For newer versions of Jotai it's highly recomended to use [`useSetAtom`](/docs/api/additional.mdx#use-set-atom) instead.
```

Comment:
It's not that "highly". In v1, we keep exporting `useUpdateAtom` from `jotai/utils`.

Reply:
Alright, makes sense. Done.